pandas/core/generic.py

```diff
@@ -2608,7 +2606,7 @@ def to_pickle(
         to_pickle(self, path, compression=compression, protocol=protocol)
 
     def to_clipboard(
-        self, excel: bool_t = True, sep: Optional[str] = None, **kwargs
+        self, excel: builtins.bool = True, sep: Optional[str] = None, **kwargs
```

Comment:
when rendering the docs, the aliased type is displayed.

![image](https://user-images.githubusercontent.com/13159005/75610915-87d7f100-5b0d-11ea-96a6-6e5b2b0f5738.png)

can you render the docs for these changes, so we can see the output.

Reply:
The docs looks the same to me:

![builtins_bool_doc](https://user-images.githubusercontent.com/50263213/75971339-4fa02c00-5eda-11ea-932d-20f810d86477.png)

Reply:
Thanks. I recall a discussion about the added "clutter" to the docs with the addition of type annotations in the function signature. (which to be fair, is redundant since we add the types to the docstring)